pkg/flag/scan_flags.go

```diff
@@ -77,44 +77,53 @@ var (
 		Default:    false,
 		Usage:      "include development dependencies in the report (supported: npm, yarn)",
 	}
+	RetainSystemInstalledFilesFlag = Flag{
+		Name:       "retain-system-installed-files",
```

Comment:
Does `pkg` make more sense?

```suggestion
		Name:       "retain-pkg-installed-files",
```